policyengine_us/parameters/gov/states/mn/tax/income/amt/fractional_income_threshold.yaml

```diff
@@ -6,23 +6,64 @@ metadata:
   breakdown:
   - filing_status
   reference:
+  - title: Minnesota Statutes Section 290.091 — Alternative Minimum Tax On Preference Items Subd.3.a
+    href: https://www.revisor.mn.gov/statutes/cite/290.091#stat.290.091.3
+  - title: 2023 Minnesota Statutes Section 270C.22 — Cost Of Living Adjustment Subdivision 1.e
+    href: https://www.revisor.mn.gov/statutes/cite/270c.22#stat.270C.22.1
   - title: 2021 Minnesota Schedule M1MT form, Line 21
     href: https://www.revenue.state.mn.us/sites/default/files/2023-02/m1mt_21.pdf#page=1
   - title: 2022 Minnesota Schedule M1MT form, Line 21
     href: https://www.revenue.state.mn.us/sites/default/files/2023-01/m1mt_22.pdf#page=1
+  - title: 2023 Minnesota Schedule M1MT form, Line 21
+    href: https://www.revenue.state.mn.us/sites/default/files/2023-12/m1mt-23.pdf#page=1
 
 JOINT:
```

Comment:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/d98bab15-f04d-48fe-9244-ec2edf312468)

Reply:
add the "values:" section to each of the breakdown